static/app/components/feedback/feedbackItem/useResolveFeedback.tsx

```diff
@@ -0,0 +1,85 @@
+import {Fragment, useCallback, useMemo} from 'react';
+
+import {
+  addErrorMessage,
+  addLoadingMessage,
+  addSuccessMessage,
+} from 'sentry/actionCreators/indicator';
+import {ModalRenderProps, openModal} from 'sentry/actionCreators/modal';
+import {Button} from 'sentry/components/button';
+import {useInfiniteFeedbackListData} from 'sentry/components/feedback/feedbackDataContext';
+import {Flex} from 'sentry/components/profiling/flex';
+import {t, tct} from 'sentry/locale';
+import {space} from 'sentry/styles/space';
+import type {HydratedFeedbackItem} from 'sentry/utils/feedback/item/types';
+import useApi from 'sentry/utils/useApi';
+import useOrganization from 'sentry/utils/useOrganization';
+import useUrlParams from 'sentry/utils/useUrlParams';
+
+interface Props {
+  feedbackItem: HydratedFeedbackItem;
+}
+
+function openResolveModal({
```

Comment:
i don't think we need the modal for archive/unarchive. It's not a permanent state, if the user wants to change back they can.